dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala

```diff
@@ -202,33 +200,25 @@ class CancelOrderTestSuite extends MatcherSuiteBase {
 
   "Auto cancel" - {
     "wrong cancel when executing a big order by small amount" in {
-      val trader = KeyPair(ByteStr("trader-auto-cancel-big-order".getBytes(StandardCharsets.UTF_8)))
-      knownAccounts = trader :: knownAccounts
-
       val amount             = 835.85722414.waves
-      val matcherFee         = 0.003.waves
       val traderTotalBalance = amount + matcherFee
-      broadcastAndAwait(mkTransfer(alice, trader, traderTotalBalance, Waves))
+      val trader             = createAccountWithBalance(traderTotalBalance -> Waves)
 
-      eventually {
-        dex1.api.tradableBalance(trader, wavesUsdPair).getOrElse(Waves, 0L) shouldBe traderTotalBalance
-      }
```

Comment:
These lines guarantee that the account has Waves. Without them the test fails on my PC. You could move it to `createAccountWithBalance` and then remove such lines in CancelOrderTestSuite.

Reply:
inside createAccountWithBalance we have broadcastAndAwait, it's enough for all other tests

Reply:
reverted this change